generated/provider_dependencies.json

```diff
@@ -38,7 +38,6 @@
       "jsonpath_ng>=1.5.3",
       "python3-saml>=1.16.0",
       "redshift_connector>=2.0.918",
-      "sqlalchemy_redshift>=0.8.6",
```

Comment:
I tried to remove it in #42830 but it seems some tests failed (but after the merge, which was really weird). We reverted it then

Reply:
Yeah I was aware and going to try poking at it again

Reply:
Cool :D Godspeed!